packages/contracts-core/.env.example

```diff
@@ -1,2 +1,3 @@
-ETHERSCAN_API_KEY=
-INFURA_API_KEY=
+export ETHERSCAN_API_KEY=
```

Comment:
`.env` files usually should not have `export` statements. if you need to pass a `.env` to a terminal command 

`env $(cat .env | grep -v "#" | xargs) some_command_here`

if you want to apply a `.env` to your shell session
`export $(cat .env | grep -v "#" | xargs)`

Reply:
That's a good catch. I got that bad habit from dealing with `dapptools` that sourced the `.env` file.